### ISSUE TITLE
files owned by a group with spaces in the name

### ISSUE THREAD

**Issue by drick78** [mimecuvalo/fireftp#139]
If a file is owned by a unix group that has spaces in the name such as "my group name", then all files that are owned by that group incorrectly show up as having the date as part of the name:

myFile.txt -> Jan 23 2014 myFile.txt

the files also have a size = 0.

**Comment by mimecuvalo** [comment]
Yeah, this is a tricky one.  The current code is here: https://github.com/mimecuvalo/fireftp/blob/master/src/content/js/connection/baseProtocol.js#L591

patches welcome! ^_^

Servers that support MLSD are much better since that is a standard format whereas the FTP 'format' is a mess :-/
